modules/examples/todolist/client/src/main/scala/clients/TodoItemClient.scala

```diff
@@ -35,4 +33,8 @@ trait TodoItemClient[F[_]] {
 
   def remove(id: Int): F[Int]
 
-}
\ No newline at end of file
+}
+
+object TodoItemClient {
+  trait Handler[G[_]] extends TodoItemClient[G]
```

Comment:
Same as in [my other comment](https://github.com/frees-io/freestyle-rpc/pull/296/files#r192694425).